src/pykiso/test_coordinator/test_case.py

```diff
@@ -227,7 +156,8 @@ def __init__(
             wait for a report during teardown execution
         :param test_ids: jama references to get the coverage
             eg: {"Component1": ["Req1", "Req2"], "Component2": ["Req3"]}
-        :param tag: dictionary containing lists of variants and/or test levels when only a subset of tests needs to be executed
+        :param tag: dictionary containing lists of variants and/or test
```

Comment:
Should be changed to something more generic:
```suggestion
        :param tag: Dictionary allowing users to filter the tests based on the keys and their value.
```